src/test/java/com/mathworks/ci/RunMatlabBuildStepTest.java

```diff
@@ -123,6 +123,18 @@ public void verifyStartupOptionsSameAsScript() throws Exception {
         j.assertLogContains("-nojvm -uniqueoption", build);
     }
 
+    /*
+     * Verify appropriate build options are invoked as in pipeline script
+     */
+    @Test
+    public void verifyBuildOptionsSameAsScript() throws Exception {
+        project.setDefinition(
+                new CpsFlowDefinition("node { runMATLABBuild(buildOptions: '-continueOnFailure -skip compile') }", true));
```

Comment:
Similarly here, good to add another test where we specify both tasks and buildOptions